packages/office-ui-fabric-react/src/components/Button/BaseButton.styles.ts

```diff
@@ -29,15 +29,19 @@ const iconStyle = {
 export const getStyles = memoizeFunction((
   theme: ITheme
 ): IButtonStyles => {
+  let { semanticColors } = theme;
+
+  let border = semanticColors.buttonBorder;
+  let disabledBackground = semanticColors.disabledBackground;
+  let disabledText = semanticColors.disabledText;
 
   return {
     root: [
       getFocusStyle(theme, -1),
       theme.fonts.medium,
       {
-        // this transparent border converts to the correct colors in HC mode
         boxSizing: 'border-box',
-        border: '1px solid transparent',
+        border: '1px solid ' + border,
```

Comment:
Can border ever be undefined or null? If so you'll need to have a check for it as javascript will concat it to '1px solidundefined'

Reply:
nope, our theming module insures it's always defined